src/univariate/continuous/rayleigh.jl

```diff
@@ -79,7 +79,7 @@ end
 
 function logccdf(d::Rayleigh, x::Real)
     z = - x^2 / (2 * d.σ^2)
-    return x > 0 ? z : isnan(x) ? oftype(z, NaN) : zero(x)
+    return x >= 0 ? z : isnan(x) ? oftype(z, NaN) : zero(x)
```

Comment:
Oh, actually we should use
```suggestion
    return x >= 0 ? z : isnan(x) ? oftype(z, NaN) : zero(z)
```
here to ensure type stability. It shouldn't matter though if we use `>` or `>=`, should it?

Reply:
Almost - it should be negative zero(z), see updated commit.